samples/client/petstore/go/go-petstore/enum_class.go

```diff
@@ -10,5 +10,12 @@
 
 package petstore
 
-type EnumClass struct {
-}
+type EnumClass string
+
+// List of EnumClasss
+const (
+	_abc EnumClass = "_abc"
```

Comment:
@marcusljx let's continue the conversation with the code here. I believe the starting "_abc" is causing the issue.

What would be the correct value looks like? "ABC"?